src/app/components/SettingsForm.tsx

```diff
@@ -47,7 +47,7 @@ function SettingsForm(props) {
           <input
             type="input"
             className="input-icon__input"
-            value={radiusValue}
+            defaultValue={props.borderRadiusValues}
```

Comment:
So I'm not sure what changed here, but upon restarting the plugin, the value that displays here should be what the user has in storage.

I just tested it by changing the values to 0,5,15, hitting save, then restarting the app and the values displayed here aren't what's stored in clientStorage. Do you mind doing a second pass on this?

Reply:
Second to this, 
<img width="322" alt="Screen Shot 2020-02-09 at 2 29 52 PM" src="https://user-images.githubusercontent.com/7717963/74111558-4aa1c400-4b4a-11ea-9a95-0ebfa9901db3.png">

I think we should force the input to have an outline. I find that Figma's default inputs don't signify they can be typed into. I'd like to leverage their design systems unless there's cases where I think it's better to break the rule.

Reply:
Yeah. Seems like it was a bug. I reverted back.